### ISSUE TITLE
Name conflict in Mammaldiet and when updating

### ISSUE THREAD

**Issue by henrykironde** [weecology/retriever#643]
Download Progress: [##############------] 67.95%https://raw.github.com/weecology/retriever/master/scripts/MammalDiet.json C:\Users\Henry\.retriever\scripts\MammalDiet.json

error
```
 check_for_updates()
  File "C:\Python27\lib\site-packages\retriever-1.9.0_de-py2.7.egg\retriever\lib\repository.py", line 52, in check_for_updates
    init.run()
  File "C:\Python27\lib\site-packages\retriever-1.9.0_de-py2.7.egg\retriever\lib\repository.py", line 125, in run
    os.path.normpath(os.path.join(SCRIPT_WRITE_PATH, script_name)))
  File "C:\Python27\lib\site-packages\retriever-1.9.0_de-py2.7.egg\retriever\lib\repository.py", line 27, in download_from_repository
    urllib.request.urlretrieve(repo + filepath, newpath)
  File "C:\Python27\lib\site-packages\future\backports\urllib\request.py", line 196, in urlretrieve
    with contextlib.closing(urlopen(url, data)) as fp:
  File "C:\Python27\lib\site-packages\future\backports\urllib\request.py", line 171, in urlopen
    return opener.open(url, data, timeout)
  File "C:\Python27\lib\site-packages\future\backports\urllib\request.py", line 500, in open
    response = meth(req, response)
  File "C:\Python27\lib\site-packages\future\backports\urllib\request.py", line 612, in http_response
    'http', request, response, code, msg, hdrs)
  File "C:\Python27\lib\site-packages\future\backports\urllib\request.py", line 532, in error
    result = self._call_chain(*args)
  File "C:\Python27\lib\site-packages\future\backports\urllib\request.py", line 466, in _call_chain
    result = func(*args)
  File "C:\Python27\lib\site-packages\future\backports\urllib\request.py", line 717, in http_error_302
    return self.parent.open(new, timeout=req.timeout)
  File "C:\Python27\lib\site-packages\future\backports\urllib\request.py", line 500, in open
    response = meth(req, response)
  File "C:\Python27\lib\site-packages\future\backports\urllib\request.py", line 612, in http_response
    'http', request, response, code, msg, hdrs)
  File "C:\Python27\lib\site-packages\future\backports\urllib\request.py", line 538, in error
    return self._call_chain(*args)
  File "C:\Python27\lib\site-packages\future\backports\urllib\request.py", line 466, in _call_chain
    result = func(*args)
  File "C:\Python27\lib\site-packages\future\backports\urllib\request.py", line 620, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
future.backports.urllib.error.HTTPError: HTTP Error 404: Not Found

```

**Comment by henrykironde** [comment]
@goelakash could you have a second look at this issue

**Comment by goelakash** [comment]
Will do.

**Comment by ethanwhite** [comment]
It's possible we fixed this yesterday in e14fd39d05bd1c80be8b1100208aa15593b3c560.

**Comment by ethanwhite** [comment]
This was fixed in e14fd39.
